README.rst

```diff
@@ -32,6 +32,17 @@ Usage is straightforward. You can substitute ``httplib2shim.Http`` anywhere ``ht
     # http is now authorized with OAuth2 credentials and uses urllib3 under
     # the covers.
 
+and for the `google-auth <https://github.com/GoogleCloudPlatform/google-auth-library-python>`_ use:
+
+.. code:: python
+
+    import httplib2shim
+    import google.auth
+    import google_auth_httplib2
+
+    credentials = google.auth.default()
```

Comment:
should be `credentials, _ = google.auth.default()`, as `default` returns the credentials and the project ID.

Reply:
Nice catch, I've amended the commit.